src/TimeService.Validation/CreateWorkTimeRequestValidator.cs

```diff
@@ -11,6 +16,37 @@ namespace LT.DigitalOffice.TimeService.Validation
 {
     public class CreateWorkTimeRequestValidator : AbstractValidator<CreateWorkTimeRequest>, ICreateWorkTimeRequestValidator
     {
+        private readonly IRequestClient<IGetUserProjectsRequest> _rcGetUserProjects;
+        private readonly ILogger<ICreateWorkTimeRequestValidator> _logger;
+
+        private bool IsUserInProject(Guid userId, Guid projectId)
+        {
+            const string logMessage = "Cannot check including user '{userId}' to project '{projectId}'.";
```

Comment:
userId and projectId words with upper case, like UserId and ProjectId. Because of code style convention for log message's arguments.

Reply:
fixed